File: Examples/Java/SDK/src/main/java/com/aspose/cells/cloud/examples/worksheet/GetComment.java
package com.aspose.cells.cloud.examples.worksheet;

import com.aspose.cells.api.CellsApi;
import com.aspose.cells.cloud.examples.Configuration;
import com.aspose.cells.cloud.examples.Utils;
import com.aspose.storage.api.StorageApi;

import java.io.IOException;
import java.nio.file.Path;

public class GetComment {

    public static void main(String... args) throws IOException {
        // ExStart: get-comment-from-worksheet
        try {
            // Instantiate Aspose Storage API SDK
            StorageApi storageApi = new StorageApi(Configuration.apiKey, Configuration.appSID, true);

            // Instantiate Aspose Words API SDK
            CellsApi cellsApi = new CellsApi(Configuration.apiKey, Configuration.appSID, true);
            String input = "Sample1.xlsx";
            Path inputFile = Utils.getPath(GetComment.class, input);
            String sheet = "Sheet1";
            String cell = "A1";

            storageApi.PutCreate(input, null, Utils.STORAGE, inputFile.toFile());

            com.aspose.cells.model.CommentResponse r = cellsApi.GetWorkSheetComment(inputFile.getFileName().toString(),
                    sheet, cell, Utils.STORAGE, null);

            System.out.println("Comment: " + r.getComment().getNote());
        }

        catch (Exception e) {
            e.printStackTrace();
        }
        // ExEnd: get-comment-from-worksheet
    }
}

File: Examples/Node.js/SDK/Worksheet/GetComment.js
// ExStart:1
var fs = require('fs');
var assert = require('assert');
var StorageApi = require('asposestoragecloud');
var CellsApi = require('asposecellscloud');
var configProps = require('../Config/config.json');
var data_path = '../../../../Data/';

var AppSID = configProps.app_sid;
var AppKey = configProps.api_key;
var config = {'appSid':AppSID,'apiKey':AppKey , 'debug' : true};

// Instantiate Aspose Storage API SDK
var storageApi = new StorageApi(config);
// Instantiate Aspose Cells API SDK
var cellsApi = new CellsApi(config);

// Set input file name
var name = "Sample_Test_Book.xls";
var sheetName = "Sheet2";
var cellName = "A4";

try {
// Upload source file to aspose cloud storage
storageApi.PutCreate(name, null, null, data_path + name , function(responseMessage) {
	assert.equal(responseMessage.status, 'OK');
	
	// Invoke Aspose.Cells Cloud SDK API to  get a specific comment from a worksheet
	cellsApi.GetWorkSheetComment(name, sheetName, cellName, null, null, function(responseMessage) {
			assert.equal(responseMessage.status, 'OK');
			var cellComment = responseMessage.body.Comment;
			console.log("Comment Note :: " + cellComment.Note);
			});
	});

}catch (e) {
  console.log("exception in example");
  console.log(e);
}
//ExEnd:1